docs/99_developer_guide.md

```
@@ -136,13 +136,15 @@ This repo currently uses the `preview` branch as a submodule at a specific commi
 Or rebuild the dev container.
 
 ### Generate documentation
-Run `doxygen` from the dev container terminal. Make sure you fix the warnings before creating a PR!  
-Generated documentation is located in the `output/docs` folder and auto-published to the `gh-pages` branch when merged with `main`.
+Run `doxygen` from the dev container terminal.  
+Generated documentation is located in the `output/api/docs`.
+Run `docfx` from the dev container terminal, The full website will now be generated in `_site/` folder.
+documentation is auto-published to the `gh-pages` branch when merged with `main`. Make sure you fix the warnings before creating a PR!
 
 Once merged, they can be viewed [here](https://polprzewodnikowy.github.io/N64FlashcartMenu/).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)